### ISSUE TITLE
Error: Failed to find duplicates

### ISSUE THREAD

**Issue by TimberWolf224** [PaulLereverend/NextcloudDuplicateFinder#63]
When opening web GUI for duplicate finder, reports "Failed to find duplicates." Browser console logs the following: 
first error: Failed to load resource: the server responded with a status of 401 (Unauthorized)
Second error: script.js?v=b74e8a3d-11:252 duplicatefinder: API Fetching TypeError: Cannot read property 'entities' of undefined
    at loadFiles (script.js?v=b74e8a3d-11:218)
    at <anonymous> Response
    
What caused the issue:
Updated nextcloud to version 22.2.3 and duplicate finder to 0.0.13. No additional changes were made.

Nextcloud log file reports no errors when the app is reloaded, though it's spammed with:
"Info | no app in context | Deprecated event type for OCP\IPreview:PreviewRequested: Symfony\Component\EventDispatcher\GenericEvent is used". No idea if this is relevant.

Webserver access log reports:
     "<clientIP> - - [timestamp] "GET /index.php/apps/duplicatefinder/api/v1/Duplicates?offset=0 HTTP/1.1" 401 43 "-" [...]"
     
I don't know that the api it's trying to reach is present. 

When the command duplicates:find-all is run, it returns duplicates as expected with no errors.
Removing and reinstalling app did not fix the issue.
Please let me know if any additional information is needed.

**Comment by chrros95** [comment]
Thanks for the report. Are you using any kind of authorization for your Nextcloud instance like basic authentication?
If yes, please add `, credentials: 'same-origin'` right at the end of https://github.com/PaulLereverend/NextcloudDuplicateFinder/blob/b6e8c302f934fcd2b8c20296c064c2e5a3b915e2/js/script.js#L214

If not, please [recorded a HAR file](https://support.zendesk.com/hc/en-us/articles/4408828867098-Generating-a-HAR-file-for-troubleshooting) that includes the first GET request.

**Comment by TimberWolf224** [comment]
Yes, I am, and the js modification fixed the issue, so thank you for your input. Perhaps worth a mention in the documentation for the project if it causes issues for other in the future.
